ee/clickhouse/queries/funnels/base.py

```diff
@@ -401,7 +401,7 @@ def _get_breakdown_conditions(self) -> str:
             if self._filter.funnel_step_breakdown:
                 values = self._parse_breakdown_prop_value()
             else:
-                limit = self._filter.breakdown_limit or 5
+                limit = self._filter.breakdown_limit or 10
```

Comment:
Aside: defaults like this might better handled in the filter object itself. :)

Reply:
I concur that this'd be even better as a `breakdown_limit_or_default` property. Can't have a default in `breakdown_limit` itself because then serialization would consider _every_ filter to have a `breakdown_limit` value, but an explicit extra property should be good.

Reply:
Makes more sense